### ISSUE TITLE
histogram tests fail on 32-bit architectures

### ISSUE THREAD

**Issue by mr-c** [pelikan-io/rustcommon#123]
`histogram` crate tests fail on 32-bit architectures, as seen in Debian

Hello, I packaged the `histogram` crate, version 0.11.0, for Debian, where we saw 2 tests fail on 32-bit architectures, such as `i386` (really `i686`). I'm opening this issue as a courtesy, as it being fixed isn't a priority for Debian: I packaged the `histogram` crate in support of a scientific computing tool which is not targeting 32-bit architectures.

### Expected behavior

Running the test from the histogram crate should pass on 32-bit x86 and other architectures.

### Actual behavior

```
561s ---- atomic::test::size stdout ----
561s thread 'atomic::test::size' panicked at src/atomic.rs:86:9:
561s assertion `left == right` failed
561s   left: 40
561s  right: 48
[…]
561s ---- standard::tests::size stdout ----
561s thread 'standard::tests::size' panicked at src/standard.rs:311:9:
561s assertion `left == right` failed
561s   left: 40
561s  right: 48
[…]
561s failures:
561s     atomic::test::size
561s     standard::tests::size
561s 
561s test result: FAILED. 19 passed; 2 failed; 1 ignored; 0 measured; 0 filtered out; finished in 0.26s
```
raw log: https://ci.debian.net/packages/r/rust-histogram/testing/i386/52816260/
archived copy of the raw log and artifacts:  [histogram-i386-log.gz](https://github.com/user-attachments/files/17503772/histogram-i386-log.gz)  [histogram-i386-artifacts.tar.gz](https://github.com/user-attachments/files/17503785/histogram-i386-artifacts.tar.gz)

### Steps to reproduce the behavior

Compile the `histogram` crate with `--target i686-unknown-linux-gnu` and run the tests.

If need be, I can cook up a `Dockerfile` to reproduce exactly how Debian does it.

**Comment by thinkingfish** [comment]
Thank you! will take a look and fix.

**Comment by brayniac** [comment]
Hi @mr-c - thanks again for reporting this. This should be addressed in today's release `histogram v0.11.1`. 

Please let us know if you encounter anything else or if that has not addressed the issue. Closing for now.
